cc_licenses/settings/base.py

```diff
@@ -210,3 +213,9 @@
 # django-distill settings
 # Put the static files in the data repo under "build"
```

Comment:
I think we can remove this comment. We are not running `collectstatic` and all static files are currently coming from the vocab/bulma cdn.